RELEASENOTES.md

```diff
@@ -7,6 +7,7 @@ Releases, starting with 9/2/2021, are listed with the most recent release at the
 __API Changes:__
 
 Added the 'META' device type, which can be used to examine the affect of shape from tensor operations without actually doing any computations.<br/>
+Added a few methods from the torch.nn.utils namespace.<br/>
 
```

Comment:
should we mention we have made the TorchSharp assembly name signed now?

Reply:
Sure. What should the comment be? Don't know assembly-name signing enough.

Reply:
Just a simple comment like: Strongly name signing the TorchSharp library to allow loading it in .NET Framework strongly named apps. 

https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named

Reply:
I'll add that to the release notes. FYI, I've taken to adding comments to the release notes with every PR I create.

Reply:
Thanks for letting me know. I am wondering how frequently we update the NuGet packages? I am asking because @michaelgsharp is waiting for the name signing change I did.

Reply:
Got it. There's no set release schedule. I make a release whenever there's a critical mass of changes, or there are just a few very important things to get out.

Reply:
We could probably make a release later this week.

Reply:
This will be great if we do it.

Reply:
Okay, let's do that, then.